lib/http.js

```diff
@@ -998,8 +999,8 @@ OutgoingRequest.prototype._start = function _start(stream, options) {
   this.headersSent = true;
 
   this.emit('socket', this.stream);
-
   var response = new IncomingResponse(this.stream);
+  response.req = this;
```

Comment:
Call this "response.request" - needlessly short names like "req" should be avoided.